packages/starknet-snap/src/sendTransaction.ts

```diff
@@ -58,15 +57,14 @@ export async function sendTransaction(params: ApiParams) {
       maxFee,
       network,
     );
-
     const response = await wallet.request({
       method: 'snap_dialog',
       params: {
         type: DialogType.Confirmation,
         content: panel([
           heading('Do you want to sign this transaction ?'),
           text(`It will be signed with address: ${senderAddress}`),
-          text(signingTxnText),
+          copyable(signingTxnText),
```

Comment:
straightly using copyable will direct output the line break text "\n\n" rather than doing a line break

we may need to divide the text into different component, e.g:
text("**heading:**"),
copyable(val),...

i have created a PR to address the layout, you can check it out and see if it is okay
https://github.com/Consensys/starknet-snap/pull/105/files